service/controller/v18/version_bundle.go

```diff
@@ -7,6 +7,11 @@ import (
 func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
+			{
+				Component:   "aws-operator",
+				Description: "Fixed a bug with removing CR before all Cloud Formation stacks are deleted.",
```

Comment:
Just saying but I find it very weird to construct sentences like this. Also `fixed` is repeated from kind.
With imperative it would be `Wait for all Cloud Formation to be deleted before removing finalizers from the CR.`.